src/dag_algo/longest_path.rs

```diff
@@ -17,7 +17,7 @@ use hashbrown::HashMap;
 use pyo3::prelude::*;
 
 use petgraph::algo;
-use petgraph::graph::NodeIndex;
+use petgraph::stable_graph::NodeIndex;
```

Comment:
Cargo fmt complains about the alphabetical order in this one

Reply:
Fixed in: https://github.com/Qiskit/rustworkx/pull/910/commits/55aeba0d4dc62b464050bbf61f1050c53d5b91a1